oleesir/authors-haven-api
# Fix Forgot and reset password

**What does this Merge request do?**

Implements ch__fixForgotAndResetPassword

**Description of Tasks completed?**

Implements ch__fixForgotAndResetPassword

**How should this be manually tested?**

- git fetch on your local machine and check out to ch__fixForgotAndResetPassword branch
- Run "npm run dev" to start the development server

**Any background context you want to provide?**

N/A


**Screenshots (if appropriate)**
